src/main/java/org/folio/marc4ld/service/marc2ld/authority/control/AuthorityIdentifierProcessorImpl.java

```diff
@@ -34,7 +35,7 @@ public void setIdentifier(Resource resource, ControlField controlField) {
     if (ObjectUtils.notEqual(controlField.getTag(), IDENTIFIER_TAG)) {
       return;
     }
-    var data = controlField.getData();
+    var data = StringUtils.deleteWhitespace(controlField.getData());
```

Comment:
"001" is a ControlField, where as "010" is a DataField.

So, we have to change the logic here as follows
```
public void setIdentifier(Resource resource, DataField dataField) {
    if (ObjectUtils.notEqual(dataField.getTag(), IDENTIFIER_TAG)) {
      return;
    }
    var lccn = dataField.getSubfield('a')
      .getData();
   // ....

}
```

Reply:
Done.